src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -63,28 +63,75 @@ public override string ToString()
     /// </summary>
     public class GraphSyncData
     {
+        /// <summary>
+        /// Deleted sub trees.
+        /// </summary>
         public List<Subtree> DeletedSubtrees
         {
             get;
             private set;
         }
 
+        /// <summary>
+        /// Added sub trees.
+        /// </summary>
         public List<Subtree> AddedSubtrees
         {
             get;
             private set;
         }
 
+        /// <summary>
+        /// Modified sub trees.
+        /// </summary>
         public List<Subtree> ModifiedSubtrees
         {
             get;
             private set;
         }
```

Comment:
Strange that these `Lists` are modified outside of `GraphSyncData` class, perhaps that needs to be modified next time.